db/schema.rb

```diff
@@ -11,15 +11,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20140710164307) do
+ActiveRecord::Schema.define(version: 20150128225725) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "issue_assignments", force: true do |t|
+  create_table "issue_assignments", force: :cascade do |t|
     t.integer  "issue_id"
-    t.datetime "created_at",                           null: false
-    t.datetime "updated_at",                           null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
I agree we should take a closer look at what's going on with the schema. It looks like some things were manually added to this file in these two commits: [1](https://github.com/codetriage/codetriage/commit/41a4b35f5252b0649a1105f962b772162e076f43) & [2](https://github.com/codetriage/codetriage/commit/9dd20530ab3b3a1176f643a658e93ceb110bdb2a).

That means when I ran my migration, it clobbered all of these edits. I don't think explicitly adding `null: false` on timestamps is necessary anymore after this [commit](https://github.com/rails/rails/commit/a939506f297b667291480f26fa32a373a18ae06a) into Rails, but I'll add back if you want. More below.